supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -32,7 +34,7 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
   override def jobExecutionActorProps(jobDescriptor: BackendJobDescriptor,
                                       initializationData: Option[BackendInitializationData],
                                       serviceRegistryActor: ActorRef): Props = {
-    JesJobExecutionActor.props(jobDescriptor, jesConfiguration, initializationData.toJes, serviceRegistryActor).withDispatcher(BackendDispatcher)
+    JesJobExecutionActor.props(jobDescriptor, jesConfiguration, initializationData.toJes.get, serviceRegistryActor).withDispatcher(BackendDispatcher)
```

Comment:
My point was more that, if we know the `.get` is fine, then it's fine to alter the method signature to reflect that.

I see they're overrides but in that case the thing calling them must be deciding based on the Option whether to call them or not?